custom/layouts/partials/usefulnessModal.ejs

```diff
@@ -0,0 +1,82 @@
+<div id="usefulness-feedback" data-document-id=<%= id %>>
+  Was this helpful?
+  <button id="usefulYes" data-launch-modal="useful-yes" class="button"><i class="fa fa-thumbs-up" aria-hidden="true"></i>Yes</button>
+  <button id="usefulNo" data-launch-modal="useful-no" class="button"><i class="fa fa-thumbs-down" aria-hidden="true"></i>No</button>
+  
+  <div class="modal" data-modal="useful-yes">
+    <div class="wrapper">
+      <section class="modalBody">
+        <div class="modalGraphic"></div>
+        <div class="modalContent">
+          <header>
+            What is the reason for your feedback?
+          </header>
+          <div><input name="reason" value="Used for growth" id="yes-reason-1" type="radio" /><label for="yes-reason-1">Used for growth</label></div>
+          <div><input name="reason" value="Used for training" id="yes-reason-2" type="radio" /><label for="yes-reason-2">Used for training</label></div>
+          <div><input name="reason" value="Other" id="yes-reason-3" type="radio" /><label for="yes-reason-3">Other</label></div>
+          <footer>
+            <button data-type="cancel" type="button" class="button">Cancel</button>
+            <button data-type="submit" type="button" class="button button-accent" value="yes">Submit</button>
+          </footer>
+        </div>
+      </section>
+    </div>
+    <button class="modal-close" data-type='cancel' type="button">Close</button>
+  </div>
+
+  <div class="modal" data-modal="useful-no">
+    <div class="wrapper">
+      <section class="modalBody">
+        <div class="modalGraphic blocky-lookng-down"></div>
+        <div class="modalContent">
+          <header>
+            What is the reason for your feedback?
+          </header>
+          <div><input name="reason" value="Not relevant" id="no-reason-1" type="radio" /><label for="no-reason-1">Not relevant</label></div>
+          <div><input name="reason" value="Hard to understand" id="no-reason-2" type="radio" /><label for="no-reason-2">Hard to understand</label></div>
+          <div><input name="reason" value="Incorrect information" id="no-reason-3" type="radio" /><label for="no-reason-3">Incorrect information</label></div>
+          <div><input name="reason" value="Other" id="no-reason-4" type="radio" /><label for="no-reason-4">Other</label></div>
+          <footer>
+            <button data-type="cancel" class="button">Cancel</button>
+            <button data-type="submit" class="button button-accent" value="no">Submit</button>
+        </div>
+      </section>
+    </div>
+    <button class="modal-close" data-type='cancel' type="button">Close</button>
+  </div>
+</div>
+<script type="text/javascript">
```

Comment:
implementing this with jquery?

Reply:
I'll wait for the build system PR and update this with the Stimulus stuff you previously saw.

Reply:
Redone with Stimulus